softvis3d-frontend/test/components/scene/Scene.spec.tsx

```diff
@@ -10,15 +10,47 @@ import SoftVis3dScene from "../../../src/components/scene/visualization/SoftVis3
 
 describe("<Scene/>", () => {
 
-    it("should initialize rendering", () => {
+    it("should initialize mounr", () => {
         let localSceneStore: SceneStore = new SceneStore();
 
-        const scene = shallow(
+        let scenePainter: SoftVis3dScene = new SoftVis3dScene();
+        localSceneStore.scenePainter = scenePainter;
+        let initStub = Sinon.stub(scenePainter, "init");
+
+        Sinon.stub(scenePainter, "getCamera", () => {
+            return {
+                position: {}
+            };
+        });
+
+        let scene = mount(
             <Scene sceneStore={localSceneStore}/>
         );
 
         expect(scene.contains(
             <SceneInformation sceneStore={localSceneStore}/>)).to.be.true;
+
+        expect(initStub.called).to.be.true;
+    });
+
+    it("should reset camera position", () => {
+        let localSceneStore: SceneStore = new SceneStore();
+
+        let scenePainter: SoftVis3dScene = new SoftVis3dScene();
+        localSceneStore.scenePainter = scenePainter;
+        let resetStub = Sinon.stub(scenePainter, "resetCameraPosition");
+
+        let underTest: Scene = new Scene();
+        underTest.props = {
+            sceneStore: localSceneStore
+        };
+
+        let event: any = {
+            keyCode: 82
```

Comment:
I did not use the static field in Scene intentionally. I want to assure that the "r" key is used for reset. The number 82 for the key r will not change. The test should be red if someone changes the number in Scene by mistake. Using the static variable would not find this.